wrappers/nodejs/src/addon.cpp

```diff
@@ -1921,54 +2168,155 @@ class RSPointCloud : public Nan::ObjectWrap {
     }
   }
 
-  static NAN_METHOD(Calculate) {
-    auto me = Nan::ObjectWrap::Unwrap<RSPointCloud>(info.Holder());
-    auto frame = Nan::ObjectWrap::Unwrap<RSFrame>(info[0]->ToObject());
-    auto target_frame = Nan::ObjectWrap::Unwrap<RSFrame>(info[1]->ToObject());
-    if (me && frame && frame->frame && target_frame) {
-      // rs2_process_frame will release the input frame, so we need to addref
-      rs2_frame_add_ref(frame->frame, &me->error);
-      rs2_process_frame(me->pc, frame->frame, &me->error);
-      auto frame = rs2_wait_for_frame(me->frame_queue, 5000, &me->error);
-      if (frame) {
-        target_frame->Replace(frame);
-        info.GetReturnValue().Set(Nan::True());
-        return;
-      }
+  static NAN_METHOD(Calculate) {
+    auto me = Nan::ObjectWrap::Unwrap<RSPointCloud>(info.Holder());
+    auto frame = Nan::ObjectWrap::Unwrap<RSFrame>(info[0]->ToObject());
+    auto target_frame = Nan::ObjectWrap::Unwrap<RSFrame>(info[1]->ToObject());
+    if (me && frame && frame->frame && target_frame) {
+      // rs2_process_frame will release the input frame, so we need to addref
+      rs2_frame_add_ref(frame->frame, &me->error);
+      rs2_process_frame(me->pc, frame->frame, &me->error);
+      auto frame = rs2_wait_for_frame(me->frame_queue, 5000, &me->error);
+      if (frame) {
+        target_frame->Replace(frame);
+        info.GetReturnValue().Set(Nan::True());
+        return;
+      }
+    }
+    info.GetReturnValue().Set(Nan::False());
+  }
+
+  static NAN_METHOD(MapTo) {
+    auto me = Nan::ObjectWrap::Unwrap<RSPointCloud>(info.Holder());
+    auto frame = Nan::ObjectWrap::Unwrap<RSFrame>(info[0]->ToObject());
+    if (me && frame) {
+      // rs2_process_frame will release the input frame, so we need to addref
+      rs2_frame_add_ref(frame->frame, &me->error);
+      rs2_process_frame(me->pc, frame->frame, &me->error);
+    }
+    info.GetReturnValue().Set(Nan::Undefined());
+  }
+
+ private:
+  static Nan::Persistent<v8::Function> constructor;
+
+  rs2_processing_block* pc;
+  rs2_frame_queue* frame_queue;
+  rs2_error* error;
+};
+
+Nan::Persistent<v8::Function> RSPointCloud::constructor;
+
+class PlaybackStatusChangedCallback :
+    public rs2_playback_status_changed_callback {
+  virtual void on_playback_status_changed(rs2_playback_status status) {
```

Comment:
nit, no public/private/protected

Reply:
Playback has not been supported, I will comment out them and add a TODO.